wsdl/ver10/schema/onvif.xsd

```diff
@@ -7967,6 +7967,10 @@ and sample rate. </xs:documentation>
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="Extension" type="tt:RecordingJobConfigurationExtension" minOccurs="0"/>
+			<xs:element name="EventFilter" type="tt:RecordingEventFilter" minOccurs="0">
+				<xs:annotation><xs:documentation>Optional filter defining on which event condition a recording job gets active.</xs:documentation></xs:annotation>
```

Comment:
Currently "tns1:RecordingHistory/Recording/State", " tns1:RecordingHistory/Track/State" events are generated by the device when recording is started or stopped. When recording is created by event trigger, where do we associate the recorded clip with the actual event so that later client can search and retrieve only event triggered recorded clips.

Reply:
Thought you were interested in actual state. For search see FindEvents and GetEventSearchResults.

Profile G has the following requirement in 7.3.1:
* Devices shall deliver notifications for Recording and Track state changes

Hence from my point of view a client can retrieve the timings of the recording snippets.